src/components/Thread.vue

```diff
@@ -304,6 +377,26 @@ export default {
 	white-space: pre-wrap;
 	user-select: text;
 }
+
+.avatar-header {
+	max-height: 24px;
+	overflow: hidden;
+}
+.avatar-more {
+	display: inline;
+	background-color: var(--color-background-dark);
+	padding: 0px 0px 1px 1px;
+	border-radius: 10px;
+	cursor: pointer;
+}
+.avatar-hidden {
+	visibility: hidden;
+}
+.popover-inner {
+	max-width: 500px;
+	border-radius: 10px;
```

Comment:
obsolete once we use the popover from nc/vue

Reply:
the `max-width` attribute still looks necessary to me

Reply:
right :)